### PR TITLE
Add round start / end triggers

### PR DESCRIPTION
**Pull request by chrisgms**
Add triggers that fire at the start / end of combat rounds. Implements #41

### CODE REVIEW

File: scripts/triggers/combat.mjs [hunk 1]
```diff
@@ -42,9 +43,10 @@ export class CombatTriggers {
     if (!combat.started || !combat.isActive) turnForward = false;
     if ((cRound < pRound) || ((cTurn < pTurn) && (cRound === pRound))) turnForward = false;
 
+    roundForward = turnForward && cRound > pRound
```
Comment (krbz999) [inline scripts/triggers/combat.mjs:46]:
```suggestion
    roundForward = turnForward && (cRound > pRound);
```

File: scripts/triggers/combat.mjs [hunk 2]
```diff
@@ -103,6 +105,13 @@ export class CombatTriggers {
       for (const c of undefeated) CombatTriggers._executeAppliedEffects(c.actor, "onEachTurn");
     }
 
+    if (roundForward) {
+      for (const c of undefeated) {
+        if (!combatStarted) CombatTriggers._executeAppliedEffects(c.actor, "onRoundEnd")
+        CombatTriggers._executeAppliedEffects(c.actor, "onRoundStart")
+      }
+    }
```
Comment (krbz999) [inline scripts/triggers/combat.mjs:113]:
```suggestion
    if (roundForward) {
      for (const c of undefeated) {
        if (!combatStarted) CombatTriggers._executeAppliedEffects(c.actor, "onRoundEnd");
        CombatTriggers._executeAppliedEffects(c.actor, "onRoundStart");
      }
    }
```

**Review by krbz999 — CHANGES_REQUESTED** [review]
Looks great, thank you for doing this. Only a few style issues from me, then I'm happy to merge.

**Review by chrisgms — COMMENTED** [review]
Merged in the styling and formatting changes

**Pull request merged**